RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZT_vect.cc

```diff
@@ -865,20 +1126,31 @@ vector<vector<reco::TransientTrack> > DAClusterizerInZT_vect::clusterize(
   
 #ifdef VI_DEBUG
   if (verbose_) {
-    std::cout  << "# DAClusterizerInZT::clusterize   pv.size=" << pv.size()
-					     << endl;
+    std::cout  << "# DAClusterizerInZT::clusterize   pv.size=" << pv.size() << endl;
   }
 #endif
 
   if (pv.empty()) {
     return clusters;
   }
-  
+
+  // fill into clusters, don't merge
+   for (auto k = pv.begin(); k != pv.end(); k++) {
+     vector<reco::TransientTrack> aCluster = k->originalTracks();
+      if (aCluster.size()>1){
+	clusters.push_back(aCluster);
+      }
+   }
+
+   return clusters;
+
+   /*
```

Comment:
Are these commented out lines needed? Either remove, or put some comment line explaining why they must remain in the code